### PR TITLE
Adds Persistent URL link to both Collection and Work views. #540

### PR DESCRIPTION
**Pull request by bwatson78**
1. catalog_controller.rb: ensures that the "id" key/value gets populated in view.
2. _find_this_item.html.erb: changes render call to the new custom partial listed below.
3. find_this_item.yml: includes id and it's label in fields for the presenter to digest.
4. _find_this_item_block.html.erb: copies from metadata block but adds logic that formats the PURL correctly.
5. spec*: adds test elements for the new metadata link.

### CODE REVIEW

**Comment by bwatson78** [comment]
Screen Shot 2020-04-29 at 10 03 36 AM

File: app/presenters/find_this_item_presenter.rb [hunk 1]
```diff
@@ -3,13 +3,18 @@
 class FindThisItemPresenter
   attr_reader :document, :config
 
-  def initialize(document:)
+  def initialize(document:, solr_document:)
```
Comment (rotated8) [inline app/presenters/find_this_item_presenter.rb:6]: Are the `document` and the `solr_document` the same?

File: app/views/catalog/_find_this_item.html.erb [hunk 2]
```diff
@@ -1 +1 @@
-<%= render 'metadata_block', document: document, presenter: FindThisItemPresenter, presenter_class: 'find-this-item', title: 'Find This Item', add_class_dt: "col-md-12", add_class_dd: "col-md-12" %>
```
Comment (rotated8) [inline app/views/catalog/_find_this_item.html.erb:1]: Is this `metadata_block` partial used elsewhere?

File: app/presenters/find_this_item_presenter.rb (hunk 1 above)
Comment (maxkadel) [reply app/presenters/find_this_item_presenter.rb:6]: Yes, I don't believe it should be needed to be added separately. I think you need to add it to the yaml file that the presenter is using... See Rachel K's two recent date PRs for a similar change. We might also want to do a refactor to make adding / changing these views a bit simpler.

Comment (bwatson78) [reply app/presenters/find_this_item_presenter.rb:6]: Nope.

File: app/views/catalog/_find_this_item.html.erb (hunk 2 above)
Comment (bwatson78) [reply app/views/catalog/_find_this_item.html.erb:1]: Yes.

**Pull request merged**